README.md

```diff
@@ -11,9 +11,10 @@ go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1
 ```
 ## proto file destination
 
-Those 2 file are used to generate go binding
-- `fig/tendermint/codec/v1/codec.proto`
-- `fig/tendermint/transform/v1/transform.proto`
+The following 2 files are used to generate Go bindings:
+- `sf/cosmos/type/v1/type.proto`
+- `sf/cosmos/transform/v1/transform.proto`
+
 This file is used to generate tendermint.ts in graph-ts and is copied to graph-node/chain/tendermint/proto/codec.proto
```

Comment:
missed that one and the line below

Reply:
> Only missed 2 in README.md.
> Don't we also want to change the name of 'graph-ts-chain/tendermint.proto'

@Seb-figment The `graph-ts` bindings are generated based on `sf/cosmos/type/v1/type.proto` ([here](https://github.com/figment-networks/tendermint-protobuf-def/blob/update-namespace/.github/workflows/release.yml#L49=)), so it seems that this file is not used. Do you think we can remove it?

Reply:
you're right. Let's remove it

Reply:
Done.

I haven't touched any other files under `graph-ts/`. I'm assuming they will be moved to a separate repository anyway.